VERSIONS.md

```diff
@@ -0,0 +1,24 @@
+# Version Lifecycle
+
+We use a modified version of [Semantic Versioning](https://semver.org/) for all changes. It is strongly encouraged that you pin at least the major version and potentially the minor version to avoid pulling in breaking changes.  Increasing the major version of an SDK indicates that this SDK underwent significant and substantial changes to support new idioms and patterns in the language. Major versions are introduced when public interfaces (e.g. classes, methods, types, etc.), behaviours, or semantics have changed. 
+
+Semantic Versions take the form of `MAJOR.MINOR.PATCH`.
+
+When bugs are fixed in the library in a backwards-compatible way, the PATCH level will be incremented by one. When new features are added to the library in a backwards-compatible way, the PATCH level will be incremented by one. PATCH changes should not break your code and are generally safe for upgrade.
```

Comment:
When new features are added to the library in a backwards-compatible way, the **MINOR** level will be incremented by one